setup/db/db/schema-452to460.sql

```diff
@@ -399,3 +399,14 @@ CREATE TABLE `cloud`.`external_bigswitch_bcf_devices` (
   CONSTRAINT `fk_external_bigswitch_bcf_devices__physical_network_id` FOREIGN KEY (`physical_network_id`) REFERENCES `physical_network`(`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `cloud`.`ldap_trust_map` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
I definitely do not want to rename domain-id to id and make it primary. its easier and convenient to have a surrogate id.
It helps me in joins and foreign keys which may be required. It will also help in normalizations or any data structure changes in futures.